force-app/main/default/lwc/productCard/productCard.js

```diff
@@ -2,43 +2,29 @@ import { LightningElement, wire } from 'lwc';
 import { CurrentPageReference } from 'lightning/navigation';
 import { NavigationMixin } from 'lightning/navigation';
 
-/** Wire adapter to load records, utils to extract values. */
-import { getRecord } from 'lightning/uiRecordApi';
+/** Utils to extract field values. */
+import { getFieldValue } from 'lightning/uiRecordApi';
 
 /** Pub-sub mechanism for sibling component communication. */
 import { registerListener, unregisterAllListeners } from 'c/pubsub';
 
 /** Product__c Schema. */
 import PRODUCT_OBJECT from '@salesforce/schema/Product__c';
 import NAME_FIELD from '@salesforce/schema/Product__c.Name';
-import LEVEL_FIELD from '@salesforce/schema/Product__c.Level__c';
```

Comment:
We should leave the static imports of field names to provide a best practice example with referential integrity, similar to [this](https://github.com/trailheadapps/lwc-recipes/blob/master/force-app/main/default/lwc/recordViewFormStaticContact/recordViewFormStaticContact.js#L10-L15). Thoughts, @kevinv11n?

Reply:
Referential integrity is recommended. Unless the example is purposely not using it (like https://github.com/trailheadapps/lwc-recipes/tree/master/force-app/main/default/lwc/recordFormDynamicContact) then it should use referential integrity. I think this example falls into the case of using it. 

`<lightning-record-view-form>` supports receiving the objects imported from `@salesforce/schema`. See https://github.com/trailheadapps/lwc-recipes/blob/master/force-app/main/default/lwc/recordViewFormStaticContact/recordViewFormStaticContact.js#L10-L15

Reply:
Thanks for the review, I added the field references.